.travis.yml

```
@@ -11,9 +11,13 @@ env:
   - LARAVEL_VERSION=5.6.*
   - LARAVEL_VERSION=5.7.*
   - LARAVEL_VERSION=5.8.*
+  - LARAVEL_VERSION=6.0.*
```

I've kept the Travis CI environment for 6.0 in the same format as the previous environment, but since Laravel 6 now follows semver, would you be open to change it to the following?
```suggestion
  - LARAVEL_VERSION=^6.0
```